libcxx/include/new

```diff
@@ -370,7 +370,6 @@ inline constexpr size_t hardware_constructive_interference_size = __GCC_CONSTRUC
 _LIBCPP_END_NAMESPACE_STD
 
 #if !defined(_LIBCPP_REMOVE_TRANSITIVE_INCLUDES) && _LIBCPP_STD_VER <= 20
-#  include <exception>
```

Comment:
It feels to me that this is one not-that-much-breaking change too many for one PR, but the workaround would be to split the header in public + private parts, where everything apart from these includes goes into private part.

What should be done here?

Reply:
This is fine.